INCHI-1-SRC/INCHI_EXE/inchi-1/src/ichimain.c

```diff
@@ -78,12 +78,12 @@
 #include "../../../INCHI_BASE/src/ichimain.h"
 #include "../../../INCHI_BASE/src/ichicomp.h"
 #include "../../../INCHI_BASE/src/ichi_io.h"
-#include "../../../INCHI_BASE/src/permutation_util.h"
 #ifdef TARGET_EXE_STANDALONE
 #include "../../../INCHI_BASE/src/inchi_api.h"
 #endif
 
 #include "../../../INCHI_BASE/src/bcf_s.h"
+#include "../../../INCHI_BASE/src/permutation_util.h"
```

Comment:
@djb-rwth, thanks for your review and suggestions.
What's the purpose of moving this include directive?
As far as I can tell, `permutation_util.h` is included unconditionally either way?
I.e., both on line 81 or line 86 the include directive isn't inside any `ifdef`?